content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -38,6 +38,35 @@ that provides the expected symbols.
 
 <!-- steps -->
 
+## Check your OS kernel version
+
+{{< tabs name="container_runtime" >}}
+{{% tab name="Linux" %}}
+
+* The Linux kernel must be 4.19 or newer, 5.x, or 6.x.
+* You can get the kernel version using the command `uname -r`
```

Comment:
this is slightly confusing -- this command should be run on the machine that will be running `kubeadm` but that's not clear here.

Reply:
This page is about installing kubeadm. I find it hard to imagine any commands should be run on a different machine than the one where you are installing kubeadm.

(I used the same wording as another section of this page, https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#verify-mac-address.)

Reply:
fair enough, if there's already precedence then its good!